envi/archs/arm/emu.py

```diff
@@ -35,19 +38,19 @@ def __init__(self):
         pass
 
     def stc(self, parms):
-        print >>sys.stderr,"CoProcEmu: stc(%s)"%repr(parms)
+        logger.info("CoProcEmu: stc(%s)"%repr(parms))
```

Comment:
logger syntax has a mechanism for passing in fmt string and args without doing the conversion in advance.
( basically, even if info() level debugging is disabled, the work to do the format string is still executing here )